docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        something like <emphasis>thor_50</emphasis> as in the above example,
+        and that would then re-read the file and spawn the 50 node Thor.
+        Caution however, if you were to put that in a helm update that would
+        do nothing, if there are no jobs running there are no Thor instances
+        running.</para>
+
+        <para>Similarly to the <emphasis>maxActive</emphasis> values in the
+        ECLCC Server, Thor has some analogous options.</para>
+
+        <para><emphasis role="bold">MaxJobs</emphasis>: controls the number of
+        jobs, specifically <emphasis>MaxJobs</emphasis> sets maximum number of
+        jobs.</para>
+
+        <para><emphasis role="bold">MaxGraphs</emphasis>: likewise
+        <emphasis>MaxGraphs</emphasis> also limits the maximum amount of
+        graphs. It makes sense to keep this value at same number, since not
+        all jobs submit graphs - but some jobs may be contained in ECL Agent
+        and may not want Thor graphs. In that case it wouldn't make sense to
+        have max of 4 If you need a Thor to generate graphs. If there are more
+        than 2 submitted (Thor) graphs, the second would be blocked until the
+        next Thor instance becomes available. It may simply make sense to set
+        <emphasis>maxGraphs</emphasis> to the same value as
+        <emphasis>MaxJobs</emphasis>.</para>
+
+        <para>The delivered (example) chart values defines maxJobs to be
+        greater than maxGraphs. It does so, because that makes sense. Jobs
+        queued to a Thor, aren't always running graphs. They might be blocked
+        on a persist, or updating super files, or waiting on some other
+        workflow event. Therefore it can make sense to have of these jobs,
+        which are not consuming a large Thor and all it's resources, but
+        restrict the max number of Thor instances running.</para>
+
+        <para>Thor has 3 components (that correspond to the resource
+        sections).</para>
+
+        <orderedlist>
+          <listitem>
+            <para>Workflow</para>
+          </listitem>
+
+          <listitem>
+            <para>Manager</para>
+          </listitem>
+
+          <listitem>
+            <para>Workers</para>
+          </listitem>
+        </orderedlist>
+
+        <para>The Manager and Workers are launched together and consume quite
+        a bit of resoures (and nodes) typically. While the Workflow is
+        inexpensive and usually doesn't require many resources. You might
+        expect in a Kubernetes world, many of them would co-exist on the same
+        node (and therefore be inexpensive). So it makes sense for maxJobs to
+        be higher, and maxGraphs to be lower</para>
+
+        <para>In containers jobs are run independently on individual pods and
+        have no overlap at all, while in bare metal we can have jobs effect
+        other (unrelated) jobs. There is another trade off with using job
```

Comment:
> while in bare metal we can have jobs effect other (unrelated) jobs

makes it sound deliberate/by design. Jobs *could* effect other jobs because they are running in the same process space.